modules/social_features/social_content_block/config/update/social_content_block_update_10200.yml

```diff
@@ -0,0 +1,10 @@
+field.field.block_content.custom_content_list.field_link:
+  expected_config: {  }
+  update_actions:
+    change:
+      translatable: TRUE
+field.field.block_content.custom_content_list.field_subtitle:
+  expected_config: {  }
+  update_actions:
+    change:
+      translatable: TRUE
```

Comment:
Need to add an empty line in the end of file here and in other places.

Reply:
fixed